tox.ini

```diff
@@ -1,7 +1,18 @@
 [tox]
-envlist = py27,py35,py36,py37,py38
+envlist = py{27,35,36,37,38}-{default,lgpl}
```

Comment:
@potiuk Missed the replacement there `use_chardet_on_py3`

Reply:
Fixed. Thanks!